dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_400ResponseServer/server.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<server description="Server for testing 400 Bad Request URI">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>servlet-4.0</feature>
+    </featureManager>
+
+    <webContainer com.ibm.ws.webcontainer.set400SCOnTooManyParentDirs="true"/>
```

Comment:
Also we should be able to set this like our other metatypes without the `com.ibm.ws.webcontainer` prefix. I think you still need some updates to WCCustomProperties to ensure both work.